packages/compass-shell/package.json

```diff
@@ -1,156 +1,157 @@
-  "karma-webpack": "^4.0.0-rc.6",
-  "less": "^2.7.2",
-  "less-loader": "^4.0.5",
-  "mini-css-extract-plugin": "^0.8.0",
-  "mocha-webpack": "^2.0.0-beta.0",
-  "mongodb-js-fmt": "^1.0.0",
-  "mongodb-js-precommit": "^2.0.0",
-  "mongodb-reflux-store": "0.0.1",
-  "node-loader": "^0.6.0",
-  "nyc": "^13.1.0",
-  "peer-deps-externals-webpack-plugin": "^1.0.2",
-  "postcss-loader": "^2.0.6",
-  "pre-commit": "^1.1.3",
-  "prop-types": "^15.7.2",
-  "re-resizable": "^6.3.2",
-  "react": "^16.8.0",
-  "react-ace": "^6.6.0",
-  "react-bootstrap": "^0.32.1",
-  "react-dom": "^16.8.0",
-  "react-hot-loader": "^4.7.1",
-  "react-redux": "^7.1.1",
-  "react-storybook-addon-chapters": "^1.1.0",
-  "react-tooltip": "^3.3.0",
-  "redux": "^4.0.4",
-  "resolve": "^1.4.0",
-  "rimraf": "^2.6.1",
-  "semver": "^5.4.1",
-  "shebang-loader": "0.0.1",
-  "sinon": "^7.2.5",
-  "sinon-chai": "^3.3.0",
-  "style-loader": "^0.18.2",
-  "url-loader": "^1.1.2",
-  "webpack": "^4.29.0",
-  "webpack-bundle-analyzer": "^3.0.3",
-  "webpack-cli": "^3.2.1",
-  "webpack-dev-server": "^3.1.14",
-  "webpack-merge": "^4.2.1",
-  "webpack-node-externals": "^1.7.2",
-  "webpack-sources": "1.0.1"
- }
+  "name": "@mongodb-js/compass-shell",
+  "productName": "Compass Shell plugin",
+  "version": "0.4.0",
+  "apiVersion": "3.0.0",
+  "description": "Compass Shell Plugin",
+  "main": "lib/index.js",
+  "scripts": {
+    "clean": "rimraf lib",
+    "precompile": "npm run clean",
+    "compile": "cross-env NODE_ENV=production webpack --config ./config/webpack.prod.config.js --progress",
+    "compile:watch": "cross-env NODE_ENV=production webpack --config ./config/webpack.prod.config.js --progress --watch",
+    "start": "cross-env NODE_ENV=development webpack-dev-server --config ./config/webpack.dev.config.js",
+    "ci": "npm run cover && npm run test:karma && npm run compile && npm run check",
+    "start:watch": "npm run clean && webpack --config ./config/webpack.watch.config.js",
+    "start-compass": "node scripts/run-in-compass.js",
+    "test": "cross-env NODE_ENV=test mocha-webpack \"./src/**/*.spec.js\"",
+    "test:watch": "cross-env NODE_ENV=test mocha-webpack \"./src/**/*.spec.js\" --watch",
+    "test:karma": "npm run prestart && cross-env NODE_ENV=test karma start",
+    "cover": "echo",
+    "fmt": "mongodb-js-fmt ./*.js ./test/*.js",
+    "check": "mongodb-js-precommit './src/**/*{.js,.jsx}' './test/**/*.js' './electron/**/*.js' './config/**/*{.js,.jsx}'",
+    "link-plugin": "./scripts/link.sh",
+    "unlink-plugin": "./scripts/unlink.sh",
+    "prepublish": "npm run compile",
+    "storybook": "start-storybook -p 9001 -c .storybook"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/mongodb-js/mongosh.git"
+  },
+  "config": {
+    "unsafe-perm": true
+  },
+  "engines": {
+    "node": ">=12.4.0"
+  },
+  "precommit": [
+    "check"
+  ],
+  "dependency-check": {
+    "ignore": [
+      "@hot-loader/react-dom",
+      "debug",
+      "acorn"
```

Comment:
sorry for the big diff in this file and package-lock.json - it was using single space and not double space for indentation. The only change in this file is adding `acorn` here to the ignore for dependency-check.